### DISCUSSION TITLE
Aqara led strip T1

### DISCUSSION THREAD

**Discussion by mcinnes01** [Koenkk/zigbee2mqtt#18690] [Devices]
Hi

I've just got the new aqara led strip T1, I've got it detected in ZigBee2mqtt but obviously as an unknown device. I've not got any experience adding a new device but would be happy to try if anyone can give me any pointers?

It also has matter support but it seems to say about needing an aqara hub which is not something I want to do. I tried adding under matter, but didn't get anywhere with that either.

Not sure where to begin, I get a lot of failed interactions like more than I've seen with any other device. This thing is RGBWW + IC so and RGB led and two white ones all individually addressable.

Cheers

Andy

**Comment by PulsarFX** [comment]
Is this still an issue? 
For me it says:

Zigbee Model
    lumi.light.acn132
Zigbee Manufacturer
    Aqara
Description
    Aqara Zigbee 3.0 LED strip T1
Support status
    Supported
